sdk/tests/integration_test.rs

```diff
@@ -52,12 +52,9 @@ async fn runner_has_token() {
     assert_lt!(balance, TokenAmount::from_whole(10000));
 }
 
-// TODO: this test fails, but it seems like it's because account deplosit is broken...
 #[tokio::test]
-#[ignore]
 async fn can_deposit() {
```

Comment:
Only enabled this one test because I know it works and can prove out the CI plumbing. We can add more tests/enable the others in subsequent PRs.